.buildscripts/deploy_snapshot.sh

```diff
@@ -1,25 +0,0 @@
-#!/bin/bash
-#
-# Deploy to Sonatype's snapshot repo.
-#
-# Adapted from https://coderwall.com/p/9b_lfq,
-# http://benlimmer.com/2013/12/26/automatically-publish-javadoc-to-gh-pages-with-travis-ci/
-# and https://github.com/square/javawriter
-
-REPO="purchases-android"
-USERNAME="RevenueCat"
-JDK="oraclejdk8"
-
-if [ "$CIRCLE_PROJECT_REPONAME" != "$REPO" ]; then
-  echo "Skipping snapshot deployment: wrong repository. Expected '$REPO' but was '$CIRCLE_PROJECT_REPONAME'."
-elif [ "$CIRCLE_PROJECT_USERNAME" != "$USERNAME" ]; then
-  echo "Skipping snapshot deployment: wrong owner. Expected '$USERNAME' but was '$CIRCLE_PROJECT_USERNAME'."
```

Comment:
I don't think we need these two conditions. With our current settings, env variables don't get sent to forks so no one would be able to do a deploy @aboedo